### DISCUSSION TITLE
Does anyone have an example of cmake + vcpkg manifest static linking?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vcpkg#37753] [Q&A] [answered]
I'm using cmake and vcpkg manifest to compile an exe that statically links qt.
I only found one example of using static linking with vcpkg and vs. Has anyone tried using vcpkg + cmake to statically link qt, or a minimal example with other libraries.

**Comment by Neumann-A** [comment, marked as answer]
Just set VCPKG_HOST_TRIPLET (and VCPKG_TARGET_TRIPLET) to a static triplet e.g. x64-windows-static-md.
